.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "conan-profiles"]
+	path = conan-profiles
+	url = https://github.com/status-im/status-conan-profiles.git
```

Comment:
Can we use `package.json` to install `conan-profiles` instead of `.gitmodules`? Both .gitmodules and "npm with package.json" seems provides the same package management functionality and mixing them increases build complexity.

Reply:
@MaxRis good point, I'll test that (see latest fixup commit)

Reply:
Done